weaviate/data/crud_data.py

```diff
@@ -438,6 +438,9 @@ def get(
         node_name: Optional[str] = None,
         consistency_level: Optional[ConsistencyLevel] = None,
         limit: Optional[int] = None,
+        offset: Optional[int] = None,
+        sort: Union[str, List[str], None] = None,
+        ascending: Union[bool, List[bool]] = True,
```

Comment:
I'm not convinced that having two different parameters to control one functionality is the best option and might be confusing for our users. Why do you think having two is better than one?

Reply:
The current implementation uses a list for sort, where sort[0] are the properties and sort[1] the order. Do you think using a dictionary would be better?

Reply:
With a `dict` the order does not matter and the `"order"` can always be omitted to have the default one (which is `asc`).